sdk/trace/batch_span_processor_test.go

```diff
@@ -257,3 +257,33 @@ func TestBatchSpanProcessorShutdown(t *testing.T) {
 	}
 	assert.Equal(t, 1, bp.shutdownCount)
 }
+
+func TestBatchSpanProcessorForceFlushSucceeds(t *testing.T) {
+	var bp testBatchExporter
+	bsp := sdktrace.NewBatchSpanProcessor(&bp)
+
+	err := bsp.ForceFlush(context.Background())
```

Comment:
To test if `ForceFlush` succeeds it should be verified that if actually flushes any held span batches not just returns a `nil` error. Can you extend this test to do that?